src/Core/System/Resources/config/loginRegistration.xml

```diff
@@ -25,6 +25,12 @@
             <label lang="de-DE">Kunden-IP-Adressen als Klartext speichern</label>
         </input-field>
 
+        <input-field type="bool">
+            <name>hideSalutation</name>
```

Comment:
The name should be the opposite instead to not differ from the other options: `showSalutation` and "Show salutation". A migration should add the value to the system config as true by default.  
Because your naming should not depend on the default value (as salutation is shown by default), but aligned with other options and what is the nicest UX.